iothub/device/src/Authentication/ClientAuthenticationWithConnectionString.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Azure.Devices.Client
     /// <summary>
     /// Authentication method that uses the device connection string to generate SAS tokens for authenticating with service.
     /// </summary>
-    public sealed class DeviceAuthenticationWithConnectionString : IAuthenticationMethod
```

Comment:
FYI, We didn't have ModuleAuthenticationWithConnectionString before, but this DeviceAuthenticationWithConnectionString only. However, this auth method is supposed to apply for both device and module, so rename it and fix the comments accordingly.